Tests/SentryTests/Protocol/SentryFrameTests.swift

```diff
@@ -7,16 +7,17 @@ class SentryFrameTests: XCTestCase {
         
         let actual = frame.serialize()
         
-        XCTAssertEqual(frame.columnNumber, actual["colno"] as? NSNumber)
+        XCTAssertEqual(frame.symbolAddress, actual["symbol_addr"] as? String)
         XCTAssertEqual(frame.fileName, actual["filename"] as? String)
         XCTAssertEqual(frame.function, actual["function"] as? String)
-        XCTAssertEqual(frame.imageAddress, actual["image_addr"] as? String)
-        XCTAssertEqual(frame.inApp, actual["in_app"] as? NSNumber)
-        XCTAssertEqual(frame.instructionAddress, actual["instruction_addr"] as? String)
-        XCTAssertEqual(frame.lineNumber, actual["lineno"] as? NSNumber)
         XCTAssertEqual(frame.module, actual["module"] as? String)
+        XCTAssertEqual(frame.lineNumber, actual["lineno"] as? NSNumber)
+        XCTAssertEqual(frame.columnNumber, actual["colno"] as? NSNumber)
         XCTAssertEqual(frame.package, actual["package"] as? String)
+        XCTAssertEqual(frame.imageAddress, actual["image_addr"] as? String)
+        XCTAssertEqual(frame.instructionAddress, actual["instruction_addr"] as? String)
         XCTAssertEqual(frame.platform, actual["platform"] as? String)
-        XCTAssertEqual(frame.symbolAddress, actual["symbol_addr"] as? String)
+        XCTAssertEqual(frame.inApp, actual["in_app"] as? NSNumber)
+        XCTAssertEqual(frame.stackStart, actual["stack_start"] as? NSNumber)
```

Comment:
I only added this line here and reordered all the other statements so they match SentryFrame.m so I could make sure all fields are here in the tests.